ex3_bod_css.css

```diff
@@ -0,0 +1,42 @@
+.textbox {
+	width: 30%;
+	margin-top: 5%;
+	margin-right: 0.5%;
+	padding: 1%;
+	border: none;
+}
+
+.textbox::placeholder{
+	border-bottom: 1px solid lightgrey;
+	color: lightgrey;
+}
+
+.textbox:focus{
+	outline: none;
+}
+
+.boxTitle {
```

Comment:
Any particular reason for using `camelCase` here? Also, what elements is this class going to be used with? I'm trying to figure out why we need the `box` prefix.